src/app/find-help/transport-search/transport-search.component.ts

```diff
@@ -70,6 +72,9 @@ export class TransportSearchComponent implements OnInit {
         this.results = results.content ?? [];
         this.total = results.totalElements ?? 0;
         this.loading = false;
+        setTimeout(() => {
```

Comment:
I am not sure but maybe after view init could eliminate this timeout or even after view cheched

Reply:
With ViewChild actually I don't need setTimeout (I removed it). With after view init, the logic is more complicated, because I don't want to scroll into view without data being fetched. How about the current version?